openfisca_uk/variables/government/dwp/pension_credit/guarantee_credit/minimum_guarantee/additional/severe_disability.py

```diff
@@ -0,0 +1,46 @@
+from openfisca_uk.model_api import *
+
+
+class severe_disability_minimum_guarantee_addition(Variable):
+    label = "Severe disability-related increase"
+    entity = BenUnit
+    definition_period = YEAR
+    value_type = float
+    unit = "currency-GBP"
+    reference = (
+        "https://www.legislation.gov.uk/uksi/2002/1792/schedule/I/paragraph/1"
+    )
+
+    def formula(benunit, period, parameters):
+        # 1. At least one adult receives a qualifying benefit
+        # 2. No children (except children receiving qualifying benefits)
+        # 3. Nobody receives Carer's Allowance (technically 'for one of the claimants', but we assume this is true)
+        severe_disability = parameters(
+            period
+        ).dwp.pension_credit.guarantee_credit.severe_disability
+        relevant_benefits = severe_disability.relevant_benefits
+        person = benunit.members
+        person_receives_qualifying_benefits = (
+            add(person, period, relevant_benefits) > 0
+        )
+        is_adult = person("is_adult", period)
+        num_eligible_adults = benunit.sum(
+            is_adult & person_receives_qualifying_benefits
+        )
+        any_children_without_benefits = (
+            benunit.sum(~is_adult & ~person_receives_qualifying_benefits) > 0
+        )
+        carers_allowance_received = (
+            benunit.sum(person("carers_allowance", period)) > 0
+        )
+        eligible = (
+            (num_eligible_adults > 0)
+            & ~any_children_without_benefits
+            & ~carers_allowance_received
+        )
+        return (
+            eligible
```

Comment:
`eligible` seems unnecessary
```suggestion
            count_eligible_adults * ~any_children_without_benefits * ~carers_allowance_received
```

Reply:
Eh I thought it seemed neater